HLTrigger/Timer/plugins/FastTimerServiceClient.cc

```diff
@@ -488,7 +492,7 @@ void FastTimerServiceClient::fillDescriptions(edm::ConfigurationDescriptions& de
   edm::ParameterSetDescription puMEPSet;
   fillPUMePSetDescription(puMEPSet);
   desc.add<edm::ParameterSetDescription>("puME", puMEPSet);
-
+  desc.add<bool>("fillEveryLumiSection", true);
```

Comment:
Shouldn't the default be "false", according to what you write in the comments?

Reply:
I think it's just that the description does not take into account the latest changes applied to the PR.

In the description, replacing this sentence
> 2. removing the per lumisection making of the histograms

with this sentence
> 2. adding an option to disable the per-lumisection updating of the histograms (which remains enabled by default)

would better reflect the content of the PR.

Reply:
@missirol in https://github.com/cms-sw/cmssw/pull/37402#issuecomment-1082655207 you were also suggesting to keep "disable per-LS updates" as the default, which is the same as written in the PR description.
@Sam-Harper , @missirol could you please agree on what's your actual intent here, and either fix the PR or its description? It can be merged right after.